modules/civiremote_event/src/Element/Confirm.php

```diff
@@ -34,7 +34,7 @@
  *
  * @FormElement("confirm")
  */
-final class Confirm extends FormElement {
+final class Confirm extends FormElementBase {
```

Comment:
`FormElementBase` was added in Drupal 10 so the code would be incompatible with earlier versions. I think this would be OK. What do you think @jensschuppe?

Reply:
Agree, Drupal 9 has been EOL for a while, so the `*.info.yml` files should be adjusted accordingly (removing compatibility with D9).